proof_transformation.ml

```diff
@@ -0,0 +1,84 @@
+open Sequent
+open Proof
+open Proof_with_notations
+open Transform_request
+
+exception Transform_exception of string;;
+
+let expand_axiom formula =
+    match formula with
+    | One -> Axiom_proof formula (* TODO apply Bottom then One ? *)
+    | Bottom -> Axiom_proof formula (* TODO apply Bottom then One ? *)
```

Comment:
@olaure01 should we apply Bottom then One ?

Reply:
Yep.

Reply:
done